.travis.yml

```diff
@@ -39,8 +53,20 @@ script:
 after_failure:
   cat v6_api/nohup.out
```

Comment:
On Travis, an API instance is started to run the tests: https://github.com/c2corg/v6_ui/blob/master/scripts/travis-api-server.sh